pkg/master/master.go

```diff
@@ -192,6 +178,20 @@ func (m *Master) InstallAPIs(c *Config) {
 		apiGroupsInfo = append(apiGroupsInfo, apiGroupInfo)
 	}
 
+	// Run the tunneler.
```

Comment:
why the move below the v1 API setup? does something in the tunneling now depend on the API or storage being available?